test/test.js

```diff
@@ -0,0 +1,141 @@
+const mocha = require('mocha')
+const { describe, it } = mocha
+
+const assert = require('assert')
+
+const windows = require('..')('1.8')
+const Item = require('prismarine-item')('1.8')
+
+const chest = windows.createWindow(1, 'minecraft:chest', 'chest', 27)
+const inv = windows.createWindow(2, 'minecraft:inventory', 'inventory')
+
+const createClick = (mode, mouseButton, slot, useInv = false) => ({
+  slot,
+  mouseButton,
+  mode,
+  windowId: useInv ? inv.id : chest.id,
+  item: slot === -999 ? null : useInv ? inv.slots[slot] : chest.slots[slot]
+})
+
+chest.updateSlot(0, new Item(1, 64))
```

Comment:
Please don't use hard coded item id's. I know this is a 1.8 specific test but item id's change between versions. It is better to use `mcData.itemsByName.dirt.id` for instance.

Reply:
implemented.